book/src/configuration-file.md

```diff
@@ -77,8 +77,12 @@ entries = [
 ```toml
 [settings]
 naptime = 0.001
+allow_erofs = false
```

Comment:
It looks like you forgot to update the documentation with the option's new name.
```suggestion
allow_remount = false
```